libc/test/src/math/CMakeLists.txt

```diff
@@ -44,6 +44,60 @@ add_fp_unittest(
     libc.src.__support.FPUtil.fp_bits
 )
 
+
+add_fp_unittest(
+  daddl_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    daddl_test.cpp
+  HDRS
+    AddTest.h
+  DEPENDS
+    libc.src.math.daddl
+)
+
+add_fp_unittest(
+  daddf128_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    daddf128_test.cpp
+  HDRS
+    AddTest.h
+  DEPENDS
+    libc.src.math.daddf128
+)
+
+add_fp_unittest(
+  ddivf128_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    ddivf128_test.cpp
+  HDRS
+    DivTest.h
+  DEPENDS
+    libc.src.math.ddivf128
+)
+
+add_fp_unittest(
+  ddivf128_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    ddivf128_test.cpp
+  HDRS
+    DivTest.h
+  DEPENDS
+    libc.src.__support.FPUtil.generic.div
+    libc.src.math.ddivf128
```

Comment:
```suggestion
  DEPENDS
    libc.src.math.ddivf128
```

Reply:
just made last minute changes if you want to check those out

Reply:
Thanks, I didn't catch that.